wisper.gemspec

```diff
@@ -17,6 +17,8 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://github.com/krisleech/wisper"
   gem.license       = "MIT"
 
+  gem.required_ruby_version = '>= 2.7'
```

Comment:
Is this strictly needed as it would mean a major version bump as it would be backwards incompatible.

Reply:
I was wondering this too. I thought the double splat is backwards compatible. @doits have you tried your MR with a lower version of Ruby?

Reply:
https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

Unless I'm missing something it should be backwards compatible.

Reply:
For ruby 2.6 you have to use `ruby2_keywords`, see https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/#a-compatible-delegation

This PR could be rewritten to use `ruby2_keywords`, then it should be compatible with ruby 2.6, too. Then, after 2.6 EOL, it must be rewritten to remove `ruby2_keywords` again (like it is done here).